go.mod

```diff
@@ -3,8 +3,12 @@ module github.com/nats-io/gnatsd
 require (
 	github.com/nats-io/go-nats v1.7.2
 	github.com/nats-io/jwt v0.2.4
+	github.com/nats-io/nats-server v1.4.1
```

Comment:
Something weird here... why go.mod of nats-server would require nats-server v.1.4.1? Also top of this file still says /nats-io/gnatsd.

Reply:
Will take a look. I removed it from vendor and made sure that worked..